### ISSUE TITLE
Outdated & missing package.json metadata

### ISSUE THREAD

**Issue by d2s** [KyleAMathews/typography.js#81]
Individual packages should have an URL to the Git repository.

[`package.json`](https://docs.npmjs.com/files/package.json#repository) files in `/packages/` directory current lack details. Missing `"repository":` field creates discovery problems for people who look individual packages at the npm package directory (as they need to search separately from search engines to find where original source code is located).

------

Also, even while individual [typography](https://www.npmjs.com/package/typography) package already has the right `"homepage":` metadata,  it has old URL in the`"repository":` field. Currently points to a broken repository address (as you had renamed repository from `typography` to `typography.js`).

_Currently:_
```
  "repository": {
    "type": "git",
    "url": "https://github.com/KyleAMathews/typography.git"
  },
```

_Should be:_
```
  "repository": {
    "type": "git",
    "url": "https://github.com/KyleAMathews/typography.js.git"
  },
```

------

Example files from another project:
- [jest: packages/jest/package.json](https://github.com/facebook/jest/blob/master/packages/jest/package.json)
- [jest: packages/jest-cli/package.json](https://github.com/facebook/jest/blob/master/packages/jest-cli/package.json)
- [jest: packages/babel-jest/package.json](https://github.com/facebook/jest/blob/master/packages/babel-jest/package.json)

------

Should individual packages's `"repository":` point to the Git repository?
(That is how Jest has solved the metadata issue.)

Do you need any help with this?

**Comment by KyleAMathews** [comment]
Hmmm... good point! This should be fixed. Just having `repository` point to this Github repo makes sense. If you want to take on creating a PR to fix this that'd be great!

**Comment by d2s** [comment]
@KyleAMathews I'll spend a bit of time to make a pull request. Review it after I post it.

**Comment by KyleAMathews** [comment]
Fixed by #83
